include/SMS/Manager/EnemyManager.hxx

```diff
@@ -44,6 +44,8 @@ public:
     void setSharedFlags();
     void updateAnmSoundShared();
 
+    const bool mIsCopyAninMtx;
```

Comment:
Can you clarify the purpose of this? Won't it shift the other members forward? It being const is very very strange as an alleged member variable.

Reply:
This was me trying to add a value defined in the code and associated with TEnemyManager. The eu map has this value, but none of the others do. It can probably be left out, adding a const variable for it was my best guess on handling these instances
![image](https://github.com/user-attachments/assets/10aeae49-3a7c-4eda-b244-359a212a139d)

Reply:
That's probably a static member variable and since it is EU only, just do a #ifdef PAL check or something along those lines. Should be easy to discover the actual name of the define.